.github/workflows/testing.yml

```diff
@@ -54,15 +54,15 @@ jobs:
       - name: Checkout
         uses: actions/checkout@master
 
-      - name: Run Unit tests
-        run: |
-          make test-unit
-
       - name: Setup Golang
         uses: actions/setup-go@v2
         with:
           go-version: '^1.15'
 
+      - name: Run Unit tests
+        run: |
+          make test-unit
```

Comment:
How did it work before without setting up Golang? :-D

Reply:
Good question; it was broken only locally when I ran it with act.